code/GraphQL/FileFilter.php

```diff
@@ -10,9 +10,20 @@
 use SilverStripe\Forms\DateField;
 use SilverStripe\ORM\ArrayList;
 use SilverStripe\ORM\Filterable;
+use SilverStripe\Dev\Deprecation;
 
+/**
+ * @deprecated 5.3.0 Will be moved to the silverstripe/graphql module in CMS 6
+ */
 class FileFilter
 {
+    public function __construct()
+    {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('5.3.0', 'Will be moved to the silverstripe/graphql module in CMS 6', Deprecation::SCOPE_CLASS);
```

Comment:
```suggestion
 * @deprecated 5.3.0 Will be moved to the silverstripe/graphql module
 */
class FileFilter
{
    public function __construct()
    {
        Deprecation::withNoReplacement(function () {
            Deprecation::notice('5.3.0', 'Will be moved to the silverstripe/graphql module', Deprecation::SCOPE_CLASS);
```

When we did the deprecations for CMS 5, we decided to be fairly standardised in the messages, and part of that was to not include a reference to the specific major release we expect the change to happen in.

Please make this change to other deprecation warnings in this PR and in the related PRs.

Reply:
Updated